app/Foliage/Meta.hs

```diff
@@ -51,10 +51,16 @@ import Toml (TomlCodec, (.=))
 import Toml qualified
 
 newtype GitHubRepo = GitHubRepo {unGitHubRepo :: Text}
-  deriving (Show, Eq, Binary, Hashable, NFData) via Text
+  deriving (Eq, Binary, Hashable, NFData) via Text
+
+instance Show GitHubRepo where
+  show = T.unpack . unGitHubRepo
```

Comment:
TBH I would prefer not to rely on the exact format of the `Show` instance. Rather I'd call `unGitHubRepo` when needed.

Reply:
Okay, I'll change it.